extending-valtimo/form-flow/whitelist-spring-bean.md

```diff
@@ -1,5 +1,10 @@
 # Whitelisting Spring beans
 
+Form flows can contain SpEL expressions. These expressions can make use of Spring beans. This page shows how Spring
```

Comment:
'Form flows can contain SpEL expressions.' -> 'It is possible to use SpEL expressions in Form flows.'